src/renderer/components/+cluster/cluster-issues.tsx

```diff
@@ -23,34 +23,38 @@ interface IWarning extends ItemObject {
   kind: string;
   message: string;
   selfLink: string;
+  age: string | number;
```

Comment:
I guess `getAge()` method from `kube-object.ts` should be refactored to avoid such a unclear types, but that is out of scope of current PR.

Reply:
I'll be sure to look into it, thanks for noticing it!